bot.py

```diff
@@ -210,6 +218,8 @@ async def townInfo(self, ctx):
     async def addTownInternal(self, ctx, post, info, message_if_exists=True):
         guild = ctx.guild
         
+        self.setToLatestVersion(guild.id)
```

Comment:
If you called addTown on a town that already existed, this would result in `!noannounce` being canceled.

Reply:
Yup, whoops.